S4TF_Tutorial_2.ipynb

```diff
@@ -210,7 +210,7 @@
         "\n",
```

Comment:
Above this, for "Of course, it would be simple enough to create a conventional Python function", should that be "Swift function"?